Contents/Sketch/styles.js

```diff
@@ -1,5 +1,14 @@
 var applyStyles = function (newStyles, sharedStyles) {
 
+  // old way to add styles: 
+  // layer.style().fills().addNewStylePart()
+  // ---->
+  // new way to add styles
+  // layer.style().addStylePartOfType(0) // To add a new fill
+  // layer.style().addStylePartOfType(1) // To add a new border
+  // layer.style().addStylePartOfType(2) // To add a new shadow
+  // layer.style().addStylePartOfType(3) // To add a new inner shadow
+
```

Comment:
Nice find. Perhaps we enumerate [like this](https://github.com/nolastan/sync.sketchplugin/blob/master/Contents/Sketch/typography.js#L3) so we don't need all the commenting. (There might be a more formal way to enumerate in cocoascript/objective-c). Not a blocker but just a suggestion for code clarity.